.npmignore

```diff
@@ -0,0 +1,23 @@
+*.log
+node_modules
+.github*
+scripts/clean-modules
```

Comment:
don't think these scripts are needed

Reply:
I moved to specifying a whitelist in package.json. This file is way too confusing.